src/shared/proto/robot_component_status.proto

```diff
@@ -0,0 +1,132 @@
+syntax = "proto2";
```

Comment:
I think I mentioned this somewhere else, but can't seem to find it now, why `proto2` over `proto3`?

Reply:
um no good reason, changed it